packages/config/config/devices/0x019b/tf016_tf021.json

```diff
@@ -223,6 +223,26 @@
 			"minValue": 50,
 			"maxValue": 400,
 			"defaultValue": 210
+		},
+		{
+			"#": "17",
+			"$if": "firmwareVersion >= 1.92",
```

Comment:
Is this really firmware 1.92 and not 1.9? The manual you linked says "1.9"

Reply:
Hi. v1.92 is the version that reads out from the device, snip from wave js us below:
![image](https://github.com/user-attachments/assets/72e351bf-c17a-46cc-b11a-f598f0532eef)
But there are no v1.91 så 1.9 would also work.

Reply:
Ok then 1.92 is fine. Seems like another case of marketing versioning.